src/main/java/net/syscon/elite/api/resource/impl/OffenderMovementsResourceImpl.java

```diff
@@ -33,4 +44,42 @@ public CourtHearing prisonToCourt(final Long bookingId, final Long courtCaseId,
     public CourtHearings getCourtHearings(final Long bookingId, final LocalDate fromDate, final LocalDate toDate) {
         return courtHearingsService.getCourtHearingsFor(bookingId, fromDate, toDate);
     }
+
+    @Override
+    public OffenderBooking moveToCell(final Long bookingId, final Long livingUnitId, final String reasonCode, final LocalDateTime dateTime) {
+        log.debug("Received moveToCell request for booking id {}, cell location {}, reasonCode {}, date/time {}",
+                bookingId,
+                livingUnitId,
+                reasonCode,
+                dateTime != null ? dateTime.format(ISO_DATE_TIME) : "null");
+
+        validateMoveToCellRequest(reasonCode);
+        final var movementDateTime = dateTime != null ? dateTime : LocalDateTime.now();
```

Comment:
I'd suggest using a Clock bean in your service for date/times.  You an then do `LocalDateTime.now(clock)` for example.

You can then take control of time in your tests.  See the `CourtHearingService` as an example.